Production_scripts/Process_extracts/extract_acute_incl_gls.R

```diff
@@ -258,52 +251,18 @@ outfile <- final_acute_file %>%
     disch,
     dischto,
     dischloc,
-    diag1,
-    diag2,
-    diag3,
-    diag4,
-    diag5,
-    diag6,
-    op1a,
-    op1b,
-    dateop1,
-    op2a,
-    op2b,
-    dateop2,
-    op3a,
-    op3b,
-    dateop3,
-    op4a,
-    op4b,
-    dateop4,
+    starts_with("diag"),
+    contains("op"),
```

Comment:
This seems very vague and could potentially match things we don't expect... How about
```suggestion
    matches("(date)?op[1-4][ab]?"),
```
I think that will work 😅

Reply:
Yeah, that is working, thanks :)